www/src/_layouts/docs.njk

```diff
@@ -20,17 +21,17 @@ styles:
       </a>
     </div>
   </nav>
-  <main class="docs__container">
+  <main class="docs__container" id="main-content">
```

Comment:
Howdy! We should probably add `tabindex="-1"` to this `<main>` to ensure focus moves here in all browsers (including mobile browsers, which tend to struggle with this kind of thing)

More context: <https://benmyers.dev/blog/skip-links/>

Reply:
More context specifically about needing to set the tabindex: https://axesslab.com/skip-links/